cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/trigger/AbstractCompositeTrigger.java

```diff
@@ -85,4 +88,14 @@ private void initializeUnitTriggers() {
       }
     }
   }
+
+  public List<TriggerInfo> getUnitTriggerInfos(TriggerInfoContext context) {
```

Comment:
huh?? That's quite confusing. I thought composite trigger is a tree of triggers, no??